swapserverrpc/server.proto

```diff
@@ -617,3 +617,42 @@ message ServerPushKeyReq {
 
 message ServerPushKeyRes {
 }
+
+service StaticAddressServer {
+    rpc ServerNewAddress (ServerNewAddressRequest)
+        returns (ServerNewAddressResponse);
+}
+
+/**
+This enum defines the protocol versions that clients may adhere to. Note that
+this is not a flagged enum. If a particular protocol version adds a feature,
+then in general all the preceding features are also supported. Exception to this
+is when features get deprecated.
+*/
+enum AddressProtocolVersion {
```

Comment:
So instead of this we could just use the existing protocol version.

Reply:
I thought we keep the address versions separated from the normal loop features but I guess new address features can just be part of the existing protocol versions.

Reply:
Done, added a new version `ProtocolVersionStaticAddress`.